### DISCUSSION TITLE
Does Supabase Accept Debit Cards?

### DISCUSSION THREAD

**Discussion by kabal1000x** [supabase/supabase#30160] [General]
Hey, does Supabase accept debit cards or only credits cards? Please do not refer me to the docs page that says "we only accept credit cards" because I have seen plenty of companies that have the same on their website and yet they do accept debits cards. Some people use the term credit cards to refer to any type of bank card. 

I could not find a way to contact Supabase sales, apparently their sales team only handles Entreprise customers. And only community help is available on Discord. I hope we can get answer from the Supabase team here.

**Comment by GaryAustin1** [comment]
https://supabase.com/dashboard/support/new Try support.
